Automattic/jetpack
# VideoPress: Input and SearchInput components

Closes #25749

#### Changes proposed in this Pull Request:

* Add `Input` and `SearchInput` components to VideoPress Package
* Fix some typescript issues

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Run `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Confirm that all stories of `Packages/VideoPress/Input` component works as expected

![Screenshot_2022-08-31_15-21-25](https://user-images.githubusercontent.com/8486249/187751398-df2236e8-9b0c-47c3-b35a-2526bee44a8f.png)
![Screenshot_2022-08-31_15-20-54](https://user-images.githubusercontent.com/8486249/187751402-5659d871-473b-4f64-8c1d-af14a3893821.png)
![Screenshot_2022-08-31_15-22-16](https://user-images.githubusercontent.com/8486249/187751596-da3205cb-babf-4430-ab76-4d8b8f835fc5.png)
![Screenshot_2022-08-31_15-22-05](https://user-images.githubusercontent.com/8486249/187751605-b7b71dcb-d65b-4233-846c-be1c1e9441ca.png)
![Screenshot_2022-08-31_17-24-27](https://user-images.githubusercontent.com/8486249/187776594-5e1ff4b8-1da5-40e8-a3c4-b8f3d04e46e6.png)
